src/coreclr/jit/gentree.cpp

```diff
@@ -8063,14 +8184,23 @@ GenTree* Compiler::gtCloneExpr(
                 goto DONE;
 
             case GT_CNS_DBL:
-                copy         = gtNewDconNode(tree->AsDblCon()->gtDconVal);
-                copy->gtType = tree->gtType; // keep the same type
+            {
```

Comment:
Nite: please follow the existing style of not using `{ }` in the function.

(Or convert it to use `{ }` everywhere)